specification/containerregistry/data-plane/Registry/readme.md

```diff
@@ -28,13 +28,23 @@ These are the global settings for the ContainerRegistry API.
 openapi-type: data-plane
 tag: package-2019-08
```

Comment:
Maybe we can take it next time, but is it okay to keep the 2019-08 as the default stable version given we already have 2021-07 version